content/flux/v0.x/join-data/troubleshoot-joins.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Troubleshoot join operations
```

Comment:
Thoughts on organizing "Issues" and "Solutions" in a table for "scan-ability"? Perhaps alphabetizing errors in a table? Recognize there's not a huge # of items, but may be nice?